save-backend/build.gradle.kts

```diff
@@ -14,12 +14,25 @@ tasks.withType<KotlinCompile> {
     }
 }
 
+tasks.getByName("check").dependsOn("copyLiquibase")
+tasks.getByName("processTestResources").dependsOn("copyLiquibase")
+
+tasks.register<Copy>("copyLiquibase") {
+    from("$rootDir/db")
+    into("$buildDir/resources/main/db")
```

Comment:
Why not `resources/test`?

Reply:
Because we have `dev` profiel instead of separate `test` resources (at least, for now).